src/main.js

```diff
@@ -0,0 +1,271 @@
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${
+                              result.dob
+                            }</dd>
+                        </div>
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key govuk-!-padding-1">Address:</dt>
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${
+                              result.addresses
+                                ? formatAddress(result.addresses[0])
+                                : ""
+                            }</dd>
+                        </div>
+                        ${
+                          result.case.status
+                            ? `
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key govuk-!-padding-1">Status:</dt>
+                            <dd class="govuk-summary-list__value govuk-!-padding-1">${formatStatus(
+                              result.case.status
+                            )}</dd>
+                        </div>
+                        `
+                            : ""
+                        }
+                    </dl>
+                </li>
+            `
+          )
+          .join("")}
+        </ul>
+        <div class="${CLASSES.item} ${CLASSES.itemSummary}">
+            Showing <strong data-id="sirius-search-summary-count">${
+              results.length
+            }</strong> of <strong>${total}</strong> results
+            <a class="govuk-link" href="/lpa/search/${searchTerm}" style="float:right">View all</a>
+        </div>
+    `);
+
+  this.resizeContainer();
+};
+
+SearchResults.prototype.render = function render(innerHTML) {
+  this.open();
+
+  this.$container.innerHTML = innerHTML;
+};
+
+SearchResults.prototype.open = function close() {
+  this.$container.hidden = false;
+};
+
+SearchResults.prototype.close = function close() {
+  this.$container.hidden = true;
+};
+
+SearchResults.prototype.resizeContainer = function resizeContainer() {
+  let hidden = 0;
+  while (this.$container.getBoundingClientRect().bottom > window.innerHeight) {
+    const items = Array.from(
+      this.$container.querySelectorAll(
+        `.${CLASSES.item}:not(.${CLASSES.itemSummary})`
+      )
+    );
+
+    if (items.length) {
+      const $remove = items[items.length - 1];
+      $remove.parentElement.removeChild($remove);
+      hidden += 1;
+    } else {
+      break;
+    }
+  }
+
+  const $total = this.$container.querySelector(
+    `.${CLASSES.itemSummary} strong:first-child`
+  );
+  if ($total instanceof HTMLElement) {
+    $total.innerText = (parseInt($total.innerText, 10) - hidden).toString();
+  }
+};
+
+document.addEventListener("submit", async (e) => {
+  const $form = e.target;
+
+  if (!($form instanceof HTMLFormElement)) return;
```

Comment:
they should never have allowed unwrapped if bodies in JS

Reply:
Switched to multi-line just for you 😁 I did check if there's a Prettier rule to enforce that but alas no

Reply:
🙏 thank you